Meta/check-license-headers.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+script_path=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+cd "$script_path/.."
+
+# We simply check if the file starts with:
+# /*
+#  * Copyright
+PATTERN=$'^/\*\n \* Copyright'
+ERRORS=()
+
+while IFS= read -r f; do
+    if [[ ! -s $f ]]; then
+        continue
+    fi
+    if [[ ! $(cat $f) =~ $PATTERN ]]; then
+        ERRORS+=($f)
+    fi
+done < <(git ls-files -- \
+'*.cpp' \
+'*.h' \
+':!:Tests' \
+':!:Base' \
+':!:Kernel/FileSystem/ext2_fs.h' \
+':!:Libraries/LibC/getopt.cpp' \
+':!:Libraries/LibCore/puff.h' \
+':!:Libraries/LibELF/exec_elf.h' \
```

Comment:
These are borrowed from somewhere and while they do contain a license header it doesn't match ours.

If someone adds a borrowed file where this is the case as well, it has to be added here manually.

Reply:
Yes.. although we should strive to get rid of these and try to avoid adding new ones :)